src/components/avatar/index.tsx

```diff
@@ -44,6 +46,35 @@ export enum BadgePosition {
 
 const DEFAULT_BADGE_SIZE = 'pimpleBig';
 
+export type NameToColorProps = {
+  /**
+   * Hash the name (or label) to get a color, so each name will have a specific color.
+   * Default is false.
+   */
+  useHashBackgroundColor: boolean;
+  /**
+   * Avatar colors to be used when useHashBackgroundColor is true
+   */
+  avatarColors?: string[];
+  /**
+   * Replace the default hashing function (name -> number)
+   */
+  hashFunction?: (name?: string) => number;
+  /**
+   * Background color in cases where the getBackgroundColor returns undefined.
+   */
+  defaultColor?: string;
```

Comment:
See my comment on `getBackgroundColor`, not sure it's still relevant

Reply:
I think we need this as a backup (errors etc)